src/lib/components/ui/input.svelte

```diff
@@ -18,11 +18,12 @@
 		controls?: boolean
 		disabled?: boolean
 		buttons?: Snippet
+		search?: boolean
 	}
 	let {
 		label,
 		labelFor = Math.random().toString(16),
-		placeholder,
+		placeholder = '',
```

Comment:
Can this stay undefined by default?
```suggestion
		placeholder,
```

Reply:
This is actually necessary as it is used to detect if user input any value with `:not(:placeholder-shown)` css rule.